deploy/upgrades/pre-caching/patch-mcv-status-processing-true.sh

```diff
@@ -0,0 +1,61 @@
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke2/managedclusterviews/view-precache-service-acct/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke5/managedclusterviews/view-precache-service-acct/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke6/managedclusterviews/view-precache-service-acct/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke1/managedclusterviews/view-precache-cluster-role-binding/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke2/managedclusterviews/view-precache-cluster-role-binding/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke5/managedclusterviews/view-precache-cluster-role-binding/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
+
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/view.open-cluster-management.io/v1beta1/namespaces/spoke6/managedclusterviews/view-precache-cluster-role-binding/status \
+--data '{"status":{"conditions":[{"lastTransitionTime":"2022-01-28T17:57:00Z","message":"found","reason":"GetResourceProcessing","status":"True","type":"Processing"}]}}'
```

Comment:
new line please

Reply:
how do you break a line in the middle of an argument string?

Reply:
Sorry I meant new line at the end of the file